clf/pipeline.go

```diff
@@ -5,6 +5,8 @@ import (
 	"strings"
 )
 
+var removeSymbolRe = regexp.MustCompile(`\W`)
```

Comment:
Another linter issue here. Best practise is to use `MustCompile`. This does the regex compile at build time thus saving processing power.